### PR TITLE
Update CCF ledger type to latest draft-ietf-cose-merkle-tree-proofs

### PR DESCRIPTION
**Pull request by ad-l**

### CODE REVIEW

File: draft-birkholz-cose-cometre-ccf-profile.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: A CoMETRE Profile and Tree Algorithm for the Confidential Consortium Framework
-abbrev: CoMTRE CFF Profile
+title: A Transaction Ledger Verifiable Structure for COSE Merkle Tree Proofs
```
Comment (henkbirkholz) [inline draft-birkholz-cose-cometre-ccf-profile.md:2]: :+1:

File: draft-birkholz-cose-cometre-ccf-profile.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+abbrev: CoMTRE CFF Ledger
```
Comment (henkbirkholz) [inline draft-birkholz-cose-cometre-ccf-profile.md:3]: Still a profile, but that does not must be up front.

File: draft-birkholz-cose-cometre-ccf-profile.md [hunk 3]
```diff
@@ -41,38 +41,39 @@ normative:
...
 {: #verifiable-data-structure-values align="left" title="Verifiable Data Structure Algorithms"}
 
-## Tree Shape
+This document defines inclusion proofs and consistency proof formats for CCF ledgers. Verifiers MUST reject all other proof types.
 
-The input of the Merkle Tree Hash (MTH) function is a list of n byte strings, written D_n = \{d\[0\], d\[1\], ..., d\[n-1\]\}. The output is a single HASH_SIZE byte string, also called the Merkle root hash.
+## Merkle Tree Shape
 
-This function is defined as follows:
+A CCF ledger is a binary Merkle Tree constructed from a hash function H, which is defined from the log type. For instance, the hash function for `CCF_LEDGER_SHA256` is `SHA256`, whose `HASH_SIZE` is 32 bytes.
```
Comment (henkbirkholz) [inline draft-birkholz-cose-cometre-ccf-profile.md:75]: Good for now. A pre-existing external references to back that description would bolster that assertion.

File: draft-birkholz-cose-cometre-ccf-profile.md [hunk 4]
```diff
@@ -41,38 +41,39 @@ normative:
...
 
-In this document, we define a new type of log, associated with the Confidential Consortium Framework (CCF) ledger. Compared to {{-certificate-transparency-v2}}, the leaves of CCF trees carry additional opaque information that is used to verify that elements are only written by the Trusted Execution Environment, which addresses the persistence of committed transactions that happen between new signatures of the Merkle Tree root.
+In this document, we define a new type of log, associated with the Confidential Consortium Framework (CCF) ledger. This log carries indexed transaction information in a binary Merkle Tree, where new transactions are appended to the right, so that the binary decomposition of the index of a transaction can be interpreted as the position in the tree if 0 represents the left branch and 1 the right branch.  Compared to {{-certificate-transparency-v2}}, the leaves of CCF trees carry additional opaque information that is used to verify that elements are only written by the Trusted Execution Environment, which addresses the persistence of committed transactions that happen between new signatures of the Merkle Tree root.
```
Comment (henkbirkholz) [inline draft-birkholz-cose-cometre-ccf-profile.md:56]: Introducing CCF would go well with a ref, see below.

File: draft-birkholz-cose-cometre-ccf-profile.md [hunk 5]
```diff
@@ -177,20 +172,13 @@ Security Considerations
...
-This document requests IANA to add the following new value to the 'COSE Header Parameters' registry:
-
-* Label: TBD_2
-* Value type: `bstr`
-* Reference: This document
-
 ### Tree Algorithms {#tree-alg-registry}
 
-This document requests IANA to add the following new value to the 'Tree Algorithms' registry:
+This document requests IANA to add the following new value to the 'COSE Verifiable Data Structures' registry:
```
Comment (henkbirkholz) [inline draft-birkholz-cose-cometre-ccf-profile.md:177]: :+1:

**Review by henkbirkholz — APPROVED** [review]
I'll push a submission based on this PR state!

**Review by OR13 — APPROVED** [review]

**Comment by ad-l** [comment]
Closing as this is superseeded by #4

**Pull request closed without merging**